### PR TITLE
PUB-1713 Updated to case name on subscription screens

### PR DESCRIPTION
**Pull request by joshblackmoor**
### JIRA link ###

https://tools.hmcts.net/jira/browse/PUB-1713

### Change description ###

Updated the subscription add from party to case name.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by KianKwa — APPROVED** [review]

File: src/main/resources/locales/en/case-name-search.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "title": "Subscribe by name of the party or parties involved",
-  "heading": "What is the name of the party or parties involved?",
+  "title": "Subscribe by case name",
```
Comment (ChrisS1512) [inline src/main/resources/locales/en/case-name-search.json:2]: Looks like the functional test is failing on case name. Probs needs updating

File: src/main/resources/locales/cy/case-name-search.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "title": "Tanysgrifo gan ddefnyddio enw'r parti neu'r partïon sydd ynglŷn â'r achos ",
-  "heading": "Beth yw enw(au)’r parti neu bartïon sy’n rhan o’r achos?",
+  "title": "Tanysgrifo gan ddefnyddio enw'r achos",
```
Comment (ChrisS1512) [inline src/main/resources/locales/cy/case-name-search.json:2]: Translations just need reverting to previous ones for now as discussed, and will bundle the requests together in the new year

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: src/main/resources/locales/en/case-name-search.json (hunk 1 above)
Comment (joshblackmoor) [reply src/main/resources/locales/en/case-name-search.json:2]: Sorted

File: src/main/resources/locales/cy/case-name-search.json (hunk 2 above)
Comment (joshblackmoor) [reply src/main/resources/locales/cy/case-name-search.json]: Done

**Review by ChrisS1512 — APPROVED** [review]

**Pull request merged**